src/plugins/opensearch_dashboards_overview/public/components/overview/overview.test.tsx

```diff
@@ -163,46 +164,42 @@ const mockFeatures = [
   },
 ];
 
-const mockBranding = {
-  darkMode: false,
-  mark: {
-    defaultUrl: '/defaultModeUrl',
-    darkModeUrl: '/darkModeUrl',
-  },
-};
+const makeProps = () => ({
+  newsFetchResult: mockNewsFetchResult,
+  solutions: mockSolutions,
+  features: mockFeatures,
+  logos: getLogosMock.default,
+});
 
 describe('Overview', () => {
-  test('render', () => {
-    const component = shallowWithIntl(
-      <Overview
-        newsFetchResult={mockNewsFetchResult}
-        solutions={mockSolutions}
-        features={mockFeatures}
-        branding={mockBranding}
-      />
-    );
-    expect(component).toMatchSnapshot();
-  });
-  test('without solutions', () => {
-    const component = shallowWithIntl(
-      <Overview
-        newsFetchResult={mockNewsFetchResult}
-        solutions={[]}
-        features={mockFeatures}
-        branding={mockBranding}
-      />
-    );
-    expect(component).toMatchSnapshot();
-  });
-  test('without features', () => {
-    const component = shallowWithIntl(
-      <Overview
-        newsFetchResult={mockNewsFetchResult}
-        solutions={mockSolutions}
-        features={[]}
-        branding={mockBranding}
-      />
-    );
-    expect(component).toMatchSnapshot();
+  describe('renders', () => {
+    it('with solutions and features', () => {
+      const props = {
+        ...makeProps(),
+      };
```

Comment:
Unnecessary memory io?

```suggestion
      const props = makeProps();
```